src/components/collection/CopyLinkButton.tsx

```diff
@@ -43,13 +44,20 @@ type CopyLinkButtonProps = { itemId: Item['id'] };
 
 const CopyLinkButton = ({ itemId }: CopyLinkButtonProps) => {
   const { t } = useLibraryTranslation();
+  const { mutations } = useContext(QueryClientContext);
 
   const { startEmbed } = useEmbedAction(itemId);
+  const { mutate: triggerAction } = mutations.usePostItemAction();
 
+  const embedItem = (e: React.MouseEvent<HTMLButtonElement>) => {
+    startEmbed(e);
+    // create an embed trigger
+    triggerAction({ itemId, payload: { type: Triggers.ItemEmbed } });
+  };
```

Comment:
What if we put this trigger action in the `useEmbedAction` hook that way you do not have to setup the action twice (in the SummaryActionButton)